### PR TITLE
Added Sponsor Page

### PR DESCRIPTION
**Pull request by GaoYeGithub**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/toronto**](https://counterspell-8t5zl8jj0.hackclub.dev/toronto) |
|:-:|
|<a href="https://counterspell-8t5zl8jj0.hackclub.dev/toronto">Screenshot of /toronto</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ScO53lxsunGQv8mjFZXbCyy6-lVnz1jLY_0R10eUjU6QJF02-wbzsBrSn9XOYEPL5ZtqX58PGLMPBrYeQiRjNw.png">(view full size)</a> |







Commit 1aa8d15eed40aad69e8977f92f87f213bdaa58cc (https://counterspell-8t5zl8jj0.hackclub.dev).

File: eslintrc.json
Comment (samdev-7) [inline eslintrc.json:1]: Please revert this addition.

File: gitignore.txt
Comment (samdev-7) [inline gitignore.txt:1]: Please revert this addition.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by GaoYeGithub** [comment]
Wait I having trouble they seem to be completely the same

**Comment by samdev-7** [comment]
You added `eslintrc.json` and `gitignore.txt`. The two files already exist as `.eslintrc.json` and `.gitignore`. Note the additional dot in front and lack of ".txt".

**Comment by GaoYeGithub** [comment]
Oh, I didn't notice thanks so much for the help

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**